tests/integration_tests/datasource_tests.py

```diff
@@ -105,6 +105,7 @@ def test_external_metadata_by_name_for_physical_table(self):
                 "database_name": tbl.database.database_name,
                 "schema_name": tbl.schema,
                 "table_name": tbl.table_name,
+                "normalize_columns": tbl.normalize_columns,
```

Comment:
Can we add some tests that create a dataset with/without this value to check for api backwards compatibility?

Reply:
Good idea, I'll do that 👍